entries/QUnit.module.xml

```diff
@@ -45,11 +63,36 @@ QUnit.module( "group b" );
 QUnit.test( "a basic test example 3", function( assert ) {
 	assert.ok( true, "this test is fine" );
 });
-QUnit.test( "a basic test example 4", function() {
+QUnit.test( "a basic test example 4", function( assert ) {
 	assert.ok( true, "this test is fine" );
 });
 ]]></code>
 	</example>
+		<example>
+		<desc>Use the <code>QUnit.module()</code> function to group tests together:</desc>
+<code><![CDATA[
+QUnit.module( "module a", function() {
+	QUnit.test( "a basic test example", function( assert ) {
+		assert.ok( true, "this test is fine" );
+	});
+});
+
+QUnit.module( "module b", function() {
+	QUnit.test( "a basic test example 2", function( assert ) {
+		assert.ok( true, "this test is fine" );
+	});
+
+	QUnit.module( "nested module b.1", function() {
+
+		// This test will be prefixed with the following module label:
+		// "module b > nested module b.1"
```

Comment:
Will it actually contain the `>`? If not, maybe just output the name of the test as it would show in the HTML reporter?